README.md

```diff
@@ -215,6 +215,44 @@ docker-compose run --rm django pytest
 ```
 
 
+## Tests End-to-End
+
+Les tests end-to-end sont écrits avec [Playwright](https://playwright.dev/).
+Les tests E2E permet de valider que les chemins utilisateurs critiques (faire une simulation, demander un avis, répondre à une demande d'avis, etc) fonctionnent correctement.
+Cela permet en outre de vérifier le bon fonctionnement des composants JavaScript de plus en plus présent sur les pages et pour le moment non couvert par d'autre tests.
+
+Ils se basent sur une base de données de tests dédiée contenant une jeu de données minimum présent dans ce [fichier](e2e/fixtures/db_seed.json)  et que l’on peut remplir pour les besoins de chaque test.
+
+Ils tournent dans la CI de Github.
+
+
+### Lancer les tests E2E en local
+Pour lancer les tests E2E en local, il faut avant tout créer une base de données dédiée similaire à celle qui sera utilisée par la CI.
+Pour cela, il faut lancer les commandes suivantes :
+
+```bash
+$ . envs/postgres
+$ docker-compose exec postgres bash -c 'dropdb envergo-test -U "$POSTGRES_USER" -f'
+$ docker-compose exec postgres bash -c 'createdb envergo-test -U "$POSTGRES_USER" -O "$POSTGRES_USER"'
+$ docker-compose run -e POSTGRES_DB=envergo-test --rm django python manage.py migrate
+$ docker compose run -e POSTGRES_DB=envergo-test --rm django python manage.py loaddata e2e/fixtures/db_seed.json
```

Comment:
```suggestion
$ docker-compose run -e POSTGRES_DB=envergo-test --rm django python manage.py loaddata e2e/fixtures/db_seed.json
```

Reply:
La V2 de Docker compose utilise la syntaxe `docker compose` et non plus `docker-compose`. 
[Je crois comprendre que dans le future c'est cette syntaxe qui devrait s'imposer ](https://docs.docker.com/compose/releases/migrate/#what-are-the-differences-between-compose-v1-and-compose-v2). Je préconise donc que l'on utilise `docker compose` partout dans les Readme. Qu'en penses tu ?

Reply:
Ok j'ignorais. Je rencontre des erreurs lorsque j'utilise cette commande telle quelle. Peut-être ajouter une mini note là dessus dans la doc ?